cpp/zotaut/validate_harvested_records.cc

```diff
@@ -42,7 +42,7 @@ namespace {
 
 
 [[noreturn]] void Usage() {
-    ::Usage("[--update-db-errors] marc_input marc_output missed_expectations_file email_address");
+    ::Usage("[--update-db-errors] marc_input marc_output online_first_files missed_expectations_file email_address");
```

Comment:
`online_first_files` should be renamed to `online_first_file` similar to `missed_expectations_file`